addons/find-bar/userscript.js

```diff
@@ -305,17 +305,25 @@ export default async function ({ addon, msg, console }) {
 
       let vars = map.getVariablesOfType("");
       for (const row of vars) {
-        addBlock(row.isLocal ? "var" : "VAR", (row.isLocal ? "var " : "VAR ") + row.name, row);
+        addBlock(
+          row.isLocal ? msg("var") + " " : msg("VAR") + " ",
+          (row.isLocal ? msg("var") + " " : msg("VAR") + " ") + row.name,
+          row
+        );
```

Comment:
`addBlock`'s first parameter is a class name, which is used to style the item and doesn't affect the displayed text. Only the second one should be translated.